src/main/features/core/desktopSettings.js

```diff
@@ -69,6 +75,10 @@ Emitter.on('settings:set', (event, details) => {
     case 'speechRecognition':
       Emitter.sendToGooglePlayMusic('speech:toggle', { state: details.value });
       break;
+    case 'startOnBoot':
+      if(details.value == true) appLauncher.enable();
```

Comment:
This can be simplified and should be written as

```js
if (details.value) {
    appLauncher.enable();
} else {
    appLauncher.disable();
}
```

I prefer readability over less lines